packages/stark-ui/src/modules/input-mask-directives/directives/text-mask-config.intf.ts

```diff
@@ -0,0 +1,46 @@
+import * as textMaskCore from "text-mask-core";
```

Comment:
Why importing everything from text-mask-core ?

Could we import only what we need like this since we provide typings ourselves ?

```typescript
import { Mask, PipeFunction } from "text-mask-core";
```

Reply:
Good point, I'm not sure if that will be posible but I'll try to apply this ;)

Reply:
It seems to work 👍